<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>homepage</class>
 <widget class="QDialog" name="homepage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profile Page</string>
  </property>
  <widget class="QLabel" name="label_welcome">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Welcome User</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>471</width>
     <height>311</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Home</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>29</y>
       <width>191</width>
       <height>181</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>189</width>
        <height>179</height>
       </rect>
      </property>
      <widget class="QTextEdit" name="textEdit_newsfeed">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>171</width>
         <height>51</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label_postInfo">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>What is on your mind?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_post">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>110</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Post</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_newsfeed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>NewsFeed</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_postArena">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Profile</string>
    </attribute>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_firstName">
         <property name="text">
          <string>First Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_Name"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_lastName">
         <property name="text">
          <string>Last Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_LastName"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_username">
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_Username"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_password">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_Password"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_gender">
         <property name="text">
          <string>Gender</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_Gender"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_birthday">
         <property name="text">
          <string>Birthday</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEdit_Birthday"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_country">
         <property name="text">
          <string>Country</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEdit_Country"/>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QLabel" name="label_updateInfo">
         <property name="text">
          <string>Note: You can only update your passoword and country</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_saveChanges">
         <property name="text">
          <string>Save Changes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_removeAccount">
         <property name="text">
          <string>Remove Account</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Search</string>
    </attribute>
    <widget class="QLabel" name="label_searchInfo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Search for people, enter their name or search by country</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_search">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_search">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QListView" name="listView_searchResult">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_sendFriend">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>150</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Send Friend Request</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Notifications</string>
    </attribute>
    <widget class="QListView" name="listView_notifications">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>256</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_notifications">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Here are your notifications</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_acceptNotification">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>230</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Accept</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ignoreNotification">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>230</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ignore</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Friends</string>
    </attribute>
    <widget class="QLabel" name="label_FriendsList">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Here are your list of friends</string>
     </property>
    </widget>
    <widget class="QListView" name="listView_friends">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_removeFriend">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>100</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Friend</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_logout">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
